### ISSUE TITLE
aria-roledescription for ruler should be "moveable"

### ISSUE THREAD

**Issue by zepumph** [phetsims/gravity-force-lab#188]
From design doc comment:

>"move in four directions is old". You can change it to "movable".

**Comment by zepumph** [comment]
Oops, this was done in https://github.com/phetsims/gravity-force-lab/issues/141
